lib/govuk_publishing_components/presenters/public_layout_helper.rb

```diff
@@ -333,7 +46,23 @@ class PublicLayoutHelper
       attr_reader :footer_navigation, :footer_meta, :cookie_banner_data
 
       def initialize(local_assigns)
-        @footer_navigation = local_assigns[:footer_navigation] || FOOTER_NAV
+        @footer_navigation = local_assigns[:footer_navigation] || I18n.t("components.layout_footer.navigation_links").each_with_index.map do |menu, i|
```

Comment:
I'm wondering what will happen if `navigation_links` in the particular translation file is empty? Will this throw up an error?

Reply:
It'll default to `en.yml`